docs/docs/documentation/user-guide/create-agents.md

```diff
@@ -14,6 +14,8 @@ This page provides an in-depth theoretical overview of creating agents in Solace
 
 Solace Agent Mesh (SAM) is a powerful platform that enables you to create intelligent agents that can communicate with each other and perform complex tasks. At its core, SAM uses a **tool-based architecture** where LLM-powered agents are equipped with specific capabilities (tools) that they can use to accomplish user requests.
 
+**Before continuing with this tutorial, it's recommend to first learn about [agent concept](../concepts/agents.md).**
```

Comment:
Before continuing with this tutorial, make sure you are familiar with the basic [agent concepts](../concepts/agents.md).